common.lic

```diff
@@ -590,6 +590,12 @@ module DRC
 
   def play_song?(settings, song_list, worn = true, skip_clean = false, climbing = false)
     instrument = worn ? settings.worn_instrument : settings.instrument
+    if UserVars.instrument != instrument
```

Comment:
Won't this force a reset for every user the first time they play a song after this is merged? Because `UserVars.instrument` will be `nil`, thus seen as a new instrument?

How about
```suggestion
    UserVars.instrument = instrument unless UserVars.instrument
    if UserVars.instrument != instrument
        message("Instrument change detected. Resetting stored song data")
        UserVars.song = nil
        UserVars.climbing_song = nil
        UserVars.instrument = instrument
    end
```